src/main/java/com/gmail/inverseconduit/javadoc/ClassInfoXmlParser.java

```diff
@@ -0,0 +1,179 @@
+package com.gmail.inverseconduit.javadoc;
+
+import java.util.Arrays;
+
+import org.w3c.dom.Document;
+import org.w3c.dom.Element;
+
+import com.gmail.inverseconduit.utils.DocumentWrapper;
+
+/**
+ * Parses {@link ClassInfo} objects from XML documents.
+ * @author Michael Angstadt
+ */
+public class ClassInfoXmlParser {
```

Comment:
most comments inside the methods look superfluous to me...
Magic strings all over the place, too. But that seems to be the simplest way here